README.md

```diff
@@ -95,12 +95,14 @@ const const const pi = 3.14!
 
 ## Naming
 
-Both variables and constants can be named with any Unicode character or string.
+Both variables and constants can be named with any Unicode character or string, except the common whitespace symbol (U+0020) and strings containing said character. Other blank characters — tabs, new lines, carriage feeds, EM spaces, and others — are apermitted.
 
 ```java
 const const firstAlphabetLetter = 'A'!
 var const 👍 = True!
 var var 1️⃣ = 1!
+const var this is 
+alt 255 = ' '!
```

Comment:
You should put a comment here that this is not using regular spaces, but (I presume) non-breakable spaces.